### PR TITLE
Tweaks to getting started

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
Will commit the screenshot next.

cc @abarth

File: getting-started.md [hunk 1]
```diff
@@ -31,8 +45,7 @@ $ git clone https://github.com/flutter/flutter.git -b alpha
 ```
 
 We update the `alpha` branch periodically as we improve Flutter. You can update
-your copy of Flutter by running `git pull` in the `flutter` directory created
-above.
+your copy of Flutter by running `flutter upgrade`.
```
Comment (abarth) [inline getting-started.md]: Do you want to be explicit about running this command inside your app?

**Comment by abarth** [comment]
LGTM

**Comment by sethladd** [comment]
Will address build failure separately: https://github.com/flutter/flutter.github.io/issues/94

**Pull request merged**